src/components/App.js

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from "react";
+import Search from "./Search";
+import SearchResults from "./SearchResults";
+
+import "../styles/app.css";
+
+const App = () => {
+  const [searchResults, setSearchResults] = useState([]);
+
+  return (
+    <div className="app">
+      <img
+        className="nasa-logo"
+        src="https://cdn.cnn.com/cnnnext/dam/assets/200424060716-nasa-worm-logo.jpg"
+        alt="nasaLogo"
```

Comment:
Alt tags are designed to be read by screen readers so they should be human readable